libraries/AP_HAL_ChibiOS/hwdef/CrazyF405/hwdef.dat

```diff
@@ -0,0 +1,134 @@
+
+# hw definition file for processing by chibios_hwdef.py
+# for BETAFLIGHTF4 hardware.
+# thanks to betaflight for pin information
+
+# MCU class and specific type
+MCU STM32F4xx STM32F405xx
+
+# board ID for firmware load
+APJ_BOARD_ID 1177
+
+# crystal frequency, setup to use external oscillator
+OSCILLATOR_HZ 8000000
+
+FLASH_SIZE_KB 1024
+
+# bootloader takes first sector
+FLASH_RESERVE_START_KB 48
+
+define HAL_STORAGE_SIZE 16384
+define STORAGE_FLASH_PAGE 1
+
+STM32_ST_USE_TIMER 5
+
+# SPI devices
+
+# SPI1
+PA5 SPI1_SCK SPI1
+PA6 SPI1_MISO SPI1
+PA7 SPI1_MOSI SPI1
+
+# SPI2
+PB13 SPI2_SCK SPI2
+PB14 SPI2_MISO SPI2
+PB15 SPI2_MOSI SPI2
+
+# SPI3
+PC10 SPI3_SCK SPI3
+PC11 SPI3_MISO SPI3
+PC12 SPI3_MOSI SPI3
+
+# Chip select pins
+PA15 FLASH1_CS CS
+PB12 OSD_CS CS
+PA4 GYRO1_CS CS
+
+# Beeper
+PB4 BUZZER OUTPUT GPIO(80) LOW
+define HAL_BUZZER_PIN 80
+
+# SERIAL ports
+SERIAL_ORDER OTG1 USART1 USART2  EMPTY EMPTY USART6
+# PA10 IO-debug-console
+PA11 OTG_FS_DM OTG1
+PA12 OTG_FS_DP OTG1
+
+# USART1
+PA10 USART1_RX USART1
+PA9 USART1_TX USART1
+
+# USART2
+PA2 USART2_TX USART2
+PA3 USART2_RX USART2
+
```

Comment:
this is the RC input
```suggestion
define DEFAULT_SERIAL3_PROTOCOL SerialProtocol_RCIN
```

Reply:
Here Serial2_protocol should RCIN right??

Reply:
copy paste error sorry....right